src/v2/BaseLoginRouter.js

```diff
@@ -109,7 +109,7 @@ export default Router.extend({
     // transform response
     const ionResponse = _.compose(
```

Comment:
why we need to use compose, bind and pass idxResponse as argument and make this complex.

I feel this kind of complexities can be avoided by simplifying it.

Reply:
I feel it's easier to understand personally. 

Otherwise it would be (guess it's more tedious when we have 3 transformers hence we have to make 3 temporary variables.

```
const tranformedResp = responseTranformer(this.settings, idpResp);
const tranformedRespWithUiSchema = uiSchemaTransform(tranformedResp);
this.appState.setIonResponse(tranformedRespWithUiSchema);
```

I wouldn't like

```
const tranformedRespWithUiSchema = uiSchemaTransform(responseTranformer(this.settings, idpResp));
this.appState.setIonResponse(tranformedRespWithUiSchema);
```

Reply:
I feel it's much easier to understand and not have to rely on maintaining the sequencing that is needed for compose. what if someone changes the sequence in _.compose ?

Reply:
> what if someone changes the sequence in _.compose

Wouldn't we worry about people change sequence in others cases?

Reply:
other's cases means ?

Reply:
@pradeepdewda-okta meaning if we write code in other ways like I posted earlier, the sequence still matters.